packages/e2e-tests/src/specs/editor/fontCheck/fontCheckMetrics.js

```diff
@@ -0,0 +1,205 @@
+
+describe('Font Check Metrics', () => {
+  let stopRequestInterception;
+  let mockResponse;
+
+  beforeAll(async () => {
+    await page.setRequestInterception(true);
+    stopRequestInterception = addRequestInterception((request) => {
+      if (request.url().includes('web-stories/v1/fonts') && mockResponse) {
+        request.respond(mockResponse);
+        return;
+      }
+
+      request.continue();
+    });
+  });
+
+  afterEach(() => {
+    mockResponse = undefined;
+  });
+
+  afterAll(() => {
+    stopRequestInterception();
+  });
+
+  it('should receive updated font metrics and not alter history', async () => {
+    const storyTitle = 'Font Check Metrics';
+    await addStoryWithFont(storyTitle);
+
+    const storyData = await getCurrentStoryData();
+
+    const fontBefore = storyData.pages[0].elements[1].font;
+    expect(fontBefore.family).toBe('Rock Salt');
+    expect(fontBefore.fallbacks).toIncludeAllMembers(['cursive']);
+    expect(fontBefore.metrics.upm).toBe(1024);
+
+    mockResponse = {
+      status: 200,
+      contentType: 'application/json',
+      body: JSON.stringify([
+        {
+          family: 'Rock Salt',
+          fallbacks: ['sans-serif'], // Original is "cursive"
+          weights: [900],
+          styles: ['italic'],
+          variants: [[0, 400]],
+          service: 'fonts.google.com',
+          metrics: {
+            upm: 200, // Original is 1024
+            asc: 1623,
+            des: -788,
+            tAsc: 824,
+            tDes: -240,
+            tLGap: 63,
+            wAsc: 1623,
+            wDes: 788,
+            xH: 833,
+            capH: 1154,
+            yMin: -787,
+            yMax: 1623,
+            hAsc: 1623,
+            hDes: -788,
+            lGap: 32,
+          },
+        },
+      ]),
+    };
+
+    await Promise.all([
+      page.waitForResponse(
+        (response) =>
+          //eslint-disable-next-line jest/no-conditional-in-test -- False positive.
+          response.url().includes('web-stories/v1/fonts') &&
+          response.status() === 200
+      ),
+      page.reload(),
+    ]);
+
+    const newStoryData = await getCurrentStoryData();
+    const fontAfter = newStoryData.pages[0].elements[1].font;
+    expect(fontAfter.family).toBe('Rock Salt');
+    expect(fontAfter.fallbacks).toIncludeAllMembers(['sans-serif']);
+    expect(fontAfter.metrics.upm).toBe(200);
+    await expect(page).toMatchElement(
+      'button[aria-label="Undo Changes"][disabled]'
+    );
+
+    await expect(page).toMatchElement(
+      'button[aria-label="Redo Changes"][disabled]'
+    );
+
+    const canUndo = await page.evaluate(
+      () =>
+        document.querySelector('button[aria-label="Undo Changes"]').disabled ===
+        false
+    );
+    await expect(canUndo).toBeFalse();
+
+    const canRedo = await page.evaluate(
+      () =>
+        document.querySelector('button[aria-label="Redo Changes"]').disabled ===
+        false
+    );
+    await expect(canRedo).toBeFalse();
```

Comment:
These seem like odd checks. Wouldn't selector above cover this?

Reply:
Yes they are indeed redundant. I think I added these originally before adding the above selector and didn't remove them afterwards.